database/seeders/DatabaseSeeder.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Database\Seeders;
+
+// use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+
+class DatabaseSeeder extends Seeder
+{
+    /**
+     * Seed the application's database.
+     */
```

Comment:
Lets use [Blumilk codestyle](https://github.com/blumilksoftware/codestyle) for some cleaning in project